scripts/continuous-integration-script.js

```diff
@@ -104,10 +104,14 @@ console.log( 'Done' );
 if ( Object.values( failedChecks ).some( checksSet => checksSet.size > 0 ) ) {
 	console.log( '\n---\n' );
 
+	console.log( `🔥 ${ RED }Errors were detected by the CI.${ NO_COLOR }\n` );
```

Comment:
```suggestion
	console.log( `🔥 ${ RED }Errors were detected by the CI.${ NO_COLOR }\n\n` );
```

I'll even add an extra line for more visibility.